zer0-os/zOS
# feat: submit selected members and add to room

### What does this do?
- submits the selected members and adds member/s to active conversation.

### Why are we making this change?
- to ensure state and client update with newly added members to room/conversation.

### How do I test this?
- Open messenger > window.FEATURE_FLAGS.enableGroupManagementMenu = true; > window.FEATURE_FLAGS.enableAddMemberToGroup = true; > Click three dot menu > Select Add Member > Add Member panel should be displayed in the sidekick (messenger list) > Add new member > Check conversation / room contains new member/s.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?

Add Members

https://github.com/zer0-os/zOS/assets/39112648/135c243f-6e97-4186-bb48-0c243817645c
